### DISCUSSION TITLE
cron job

### DISCUSSION THREAD

**Discussion by storm-beyndtech** [vercel/community#1859] [Help] [answered]
### Summary

my cron job is not working automaticall again but when i click run in the console it works fine

**Comment by amyegan** [comment]
Thanks for reaching out about this, @khameleonstorm. If there is any more information about your setup that you can share, please let us know to help with our investigation 🙏

**Comment by amyegan** [comment, marked as answer]
The engineering team identified the cause of the problem and confirmed that Cron invocations have resumed. The missed Cron invocations will not be retried to avoid causing issues on your projects, but Cron Jobs are back to normal for future invocations

**Reply by storm-beyndtech** [reply]
it stopped again yesterday and i'm loosing clients

**Reply by amyegan** [reply]
@khameleonstorm I haven't seen any other reports of this being broken again, so it may be a different problem this time. I have a few questions to help guide us in the right direction so we can solve this as quickly as possible.

- Is it behaving the same as it did last week, where clicking run in the console worked fine?
- Did it start working again after Cron invocations were supposed to have resumed? Or has it remained broken for you the whole time?
- Were any changes made to your cron config or serverless function before it broke?

**Reply by jesper-bylund** [reply]
I have the same issue. You need to trigger an alert and a manual override if any crashed cron job will never be retried
